CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+## 0.7.0-SNAPSHOT (current master)
+
+* integrate [ohsome-filter](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter) module fully into this repository, renaming it to `oshdb-filter`. #306
```

Comment:
Would there be transition documentation about how to further use the filter functionality? Like when changing the oshdb groupId:
> When switching to the OSHDB version 0.6 you need to adapt your `pom.xml` to the new groupId, e.g.:
> ```xml
> <dependency>
>   <groupId>org.heigit.ohsome</groupId>
>   <artifactId>oshdb-api</artifactId>
>   <version>0.6.0</version>
> </dependency>
> ```

Reply:
true, let me add that

Reply:
well… actually, nothing much changed. Only when one used the `filter(FilterExpression)` method (e.g. via `.filter(Filter.byOSHEntity(…))` one would need to update the package name in the import. Otherwise everything would stay the same. Or am I overlooking something here?

Reply:
I think you have better insight. It was more meant like a question if you want to add something like that here or not.